packages/nextjs/app/debug/_components/contract/utilsContract.tsx

```diff
@@ -19,7 +19,7 @@ const isJsonString = (str: string) => {
 };
 
 const isBigInt = (str: string) => {
-  if (str.trim().length === 0) return false;
+  if (str.trim().length === 0 || str.startsWith("0")) return false;
```

Comment:
Umm not sure whats the best way, but this seems to work fine. 

The reason I am not doing `startsWith("0x")` and used only "0", because if you keep `startsWith("0x")` then this happens with string starting with 0, notice how it ignores 0 in input string


https://github.com/user-attachments/assets/958346f5-acda-475a-b15a-84917c437f57